extensions/wikia/Recirculation/services/CurationCmsArticleService.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+use GuzzleHttp\Client;
+use GuzzleHttp\Exception\ClientException;
+use Psr\Http\Message\ResponseInterface;
+use Wikia\Factory\ServiceFactory;
+use Wikia\Logger\WikiaLogger;
+use function GuzzleHttp\Psr7\build_query;
+
+class CurationCmsArticleService implements FandomArticleService {
+	const FAILED_TO_CALL_CURATION_CMS = "Failed to call curation-cms";
+	const FANDOM_SITE_NAME = 'Fandom';
+
+	const REQUEST_TIMEOUT_SECONDS = 2;
+	const EXTRA_POSTS_TO_FETCH = 20;
+
+	private const FANDOM_BASE_URL = 'https://www.fandom.com';
+
+	/** @var string */
+	private $baseUrl;
+
+	public function __construct() {
+		$urlProvider = ServiceFactory::instance()->providerFactory()->urlProvider();
+		$this->baseUrl = 'http://' . $urlProvider->getUrl( 'curation-cms' ) . '/stories/feed/articles';
+	}
+
+	/**
+	 * @param $limit
+	 * @return ResponseInterface
+	 * @throws WikiaException
+	 */
+	private function doApiRequest( int $limit ): ResponseInterface {
+		$client = new Client( [
+			// Base URI is used with relative requests
+			'base_uri' => $this->baseUrl,
+			// You can set any number of default request options.
+			'timeout' => 2.0,
```

Comment:
Why not use `REQUEST_TIMEOUT_SECONDS`? It's unused otherwise.

Reply:
https://github.com/Wikia/app/pull/17629/commits/77fb67ddbc0b0a81f338056dddca39e8e87db161